### PR TITLE
Getting Started Article - Discord.py added

### PR DESCRIPTION
**Pull request by Aman-Clement**

### CODE REVIEW

File: _gettingstarted/discord.md [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    <img src="../assets/images/articles/discord.py/reset-token.png" alt="Reset Token" style="width: 70%; margin: 10px">
+</div>
+> ##### Note
+>Do not share this token with anyone. Exposure of this token could lead to unauthorized access and compromise of sensitive information
+
+Now that we are all set, let us dive right into building our discord bot.
+
+First and foremost let us install discord.py
+
+```python
```
Comment (ayushjain01) [inline _gettingstarted/discord.md]: this should be bash not python

File: _gettingstarted/discord.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+python3 -m pip install -U discord.py
+```
+
+To get voice support also, we should run : 
+```python
```
Comment (ayushjain01) [inline _gettingstarted/discord.md]: same here

File: _gettingstarted/discord.md [hunk 3]
```diff
@@ -0,0 +1,341 @@
...
+
+bot.add_command(test)
+```
+> ##### Note:
+> Any parameter that is accepted by the Command constructor can be passed into the decorator. For example, to change the name to something other than the function would be as simple as doing this
+> ```python
+>@client.command(name='list')
+>async def _list(ctx, arg):
+>    pass
+```
```
Comment (ayushjain01) [inline _gettingstarted/discord.md]: Add > before ```

File: _gettingstarted/discord.md [hunk 4]
```diff
@@ -0,0 +1,341 @@
...
+
+## Cogs
+As the number of commands increase, your main.py becomes huge and it becomes inefficient and ugly to manage the bot. Cogs is a concept of grouping commands with a similar concept or functioning for a common goal, for example, all the moderation commands can be grouped together, all the commands related to voice channels can be grouped together in another file. Now we will see how to define and sync commands after grouping them.
+
+First let us create a folder cogs and create the categories.
+<div style="display: flex; justify-content: center;">
+    <img src="../assets/images/articles/discord.py/cogs.png" alt="sidebar" style="width: 70%; margin: 10px">
+</div>
+> your categories may differ based on your requirement, this is just defining the structure
```
Comment (ayushjain01) [inline _gettingstarted/discord.md:184]: Note*

File: _gettingstarted/discord.md [hunk 5]
```diff
@@ -0,0 +1,341 @@
...
+            await ctx.send('You do not have permissions for that lil homie')
+
+async def setup(client):
+    await client.add_cog(Moderation(client)) 
+```
+Here we have defined moderation commands to moderate the users. Kick command is used to kick a user from the server, the command first checks if the user calling this command has the authority to kick the specified user, MissingPermissions handles this.
+Ban command is similar to kick, but once a user is banned he cannot join back unless they are unbanned, both of these commands check the permissions of the person calling it.
+
+## Advanced Application
+In this article we have laid the basic foundation for the discord bot, but the possibilities are endless, we can work with voice commands, various decorators, error handling, manage members. Discord.py has a very large active community to help you and has a very good documentation to help you achieve your end goal.
```
Comment (ayushjain01) [inline _gettingstarted/discord.md:342]: You can add faqs and related articles within the same article

**Review by ayushjain01 — CHANGES_REQUESTED** [review]
Article looks good, only some minor formatting issues

**Review by tankala — APPROVED** [review]

**Pull request merged**